mdm/orientdb-wrapper/src/main/java/com/gentics/mesh/core/data/node/impl/NodeImpl.java

```diff
@@ -250,7 +250,9 @@ public Stream<Node> getChildrenStream(InternalActionContext ac, InternalPermissi
 		Tx tx = GraphDBTx.getGraphTx();
 		UserDao userDao = tx.userDao();
 		return toStream(getUnframedChildren(tx.getBranch(ac).getUuid()))
-			.filter(node -> userDao.hasPermissionForId(user, node.getId(), perm))
+			.filter(node -> perm != null 
```

Comment:
Fallback to the old behavior when no perm parameter provided.

Reply:
This does not make any sense. The method is never called without perm from productive code, only from the testcase NodeTest.testGetChildrenStream(), so it is better to fix that testcase.

Reply:
Testcase has now been fixed in dev branch.